pygmt/tests/test_info.py

```diff
@@ -13,6 +13,13 @@
 from pygmt import info
 from pygmt.exceptions import GMTInvalidInput
 
+try:
+    import pyarrow  # noqa: F401
+
+    HAS_PYARROW = True
+except ImportError:
+    HAS_PYARROW = False
+
```

Comment:
Since `pyarrow` is not directly used, we can follow the suggestions of the `F401` rule (https://docs.astral.sh/ruff/rules/unused-import/) and use `importlib.util.find_spec`:

```
from importlib.util import find_spec

if find_spec("pyarrow") is not None:
    HAS_PYARROW = True
else:
    HAS_PYARROW = False
```

Reply:
That's ok, we won't need this try-except or if-else block once we have the `skip_if_no` helper function at #2883.